### PR TITLE
Implement real time search for adding users to Trip

### PR DESCRIPTION
**Pull request by NeHerkus**

### CODE REVIEW

File: src/app/user-search/user-search.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  results: Person[];
+  queryField: FormControl = new FormControl();
+
+  constructor(private searchService: UserSearchService) {
+  }
+
+  ngOnInit() {
+    this.queryField.valueChanges.pipe(debounceTime(400),
+      distinctUntilChanged(), switchMap((query) => {
+          if (query !== '') {
```
Comment (PovilasKM) [inline src/app/user-search/user-search.component.ts]: o kas jeigu įvesiu tarpelį?

File: src/app/user-search/user-search.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div>
+  <label for="formControlSearch">Add people</label>
+  <input [formControl]="queryField"
```
Comment (maaarsas) [inline src/app/user-search/user-search.component.html]: Maybe we should use this:
https://ng-bootstrap.github.io/#/components/typeahead/examples
Nicer and with many features

File: src/app/user-search/user-search.component.ts (hunk 1 above)
Comment (maaarsas) [reply src/app/user-search/user-search.component.ts]: Ieškos pagal tarpelį? :D

**Comment by maaarsas** [comment]
Pataisyk ir sumergeinsim? https://travis-ci.com/maaarsas/psk-trip-planner-web/builds/107367961

File: src/app/planning/trip-create/trip-create.component.ts [hunk 3]
```diff
@@ -7,7 +7,8 @@ import { Component, OnInit } from '@angular/core';
 })
 export class TripCreateComponent implements OnInit {
 
-  constructor() { }
+  constructor() {
+  }
```
Comment (gpiliponyte) [inline src/app/planning/trip-create/trip-create.component.ts]: wow

Comment (maaarsas) [reply src/app/planning/trip-create/trip-create.component.ts]: 🥇

**Pull request merged**